terminus-ui/table/src/cell.ts

```diff
@@ -100,44 +119,64 @@ export class TsHeaderCellDirective extends CdkHeaderCell {
   },
 })
 export class TsCellDirective extends CdkCell {
+  public column: TsColumnDefDirective;
+
   constructor(
-    columnDef: CdkColumnDef,
-    elementRef: ElementRef,
-    public renderer: Renderer2,
+    private columnDef: CdkColumnDef,
+    private elementRef: ElementRef,
+    private renderer: Renderer2,
+    @Optional() @Host() @SkipSelf() parent: TsTableComponent<unknown>,
   ) {
     super(columnDef, elementRef);
-
-    // NOTE: We are adding `noWrap` to the column in `TsColumnDefDirective` which doesn't exist
-    // in the `CdkColumnDef` so we cast it to 'any'.
-    // tslint:disable-next-line no-any
-    const column: any = columnDef;
+    // NOTE: Changing the type in the constructor from `CdkColumnDef` to `TsColumnDefDirective` doesn't seem to play well with the CDK.
+    // Coercing the type here is a hack, but allows us to reference properties that do not exist on the underlying `CdkColumnDef`.
+    this.column = columnDef as TsColumnDefDirective;
```

Comment:
can `columnDef` be defined as `TsColumnDefDirective`? oh I guess you'd need to use some `CdkColumnDef` built in method

Reply:
I honestly thought it could since mine is an extension of theirs, but everything broke when I did. This doesn't feel great, but I was trying to keep moving and technically `columnDef` _is_ `TsColumnDefDirective` so I don't think I'm cheating _too_ much